app/common/schema.ts

```diff
@@ -4,7 +4,7 @@ import { GristObjCode } from "app/plugin/GristData";
 
 // tslint:disable:object-literal-key-quotes
 
-export const SCHEMA_VERSION = 41;
+export const SCHEMA_VERSION = 43;
```

Comment:
Any reason to jump from version 41 to 43 ?

Reply:
I have done an 42 that was wrong and locally my data base could not understand the migrations when I pass this 43 to 42 ... I will try one more time to have it cleaner